tools/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "1.0.0",
   "description": "Generates release notes",
   "main": "index.js",
+  "type": "module",
```

Comment:
not sure why you need to use 'module' there ? (and generate a .cjs file instead of a .js file)

you want to avoid the use of a vite.config.js file ?